corehq/apps/sso/templates/sso/email/api_secret_expiring_reminder.html

```diff
@@ -0,0 +1,34 @@
+{% load i18n %}
+<p>{% blocktrans %}Dear enterprise administrator,{% endblocktrans %}</p>
+
+<p>
+  {% blocktrans %}
+  The api secret for your Identity Provider {{ idp_name }} expires soon!
+  If the secret is not recreated with a new expiration date, the auto-deactivation of web user for this IdP will stop working {{ expires_on }}
+  {% endblocktrans %}
+</p>
+
+<p>
+  {% blocktrans %}
+    To change or renew your subscription, please reach out to {{ sales_email }}.
+  {% endblocktrans %}
+</p>
+
+<p>
+  {% blocktrans %}
+    If you have any questions, please don’t hesitate to contact
+    {{ contact_email }}. Thank you for your use and support of CommCare.
+  {% endblocktrans %}
+</p>
+
+
+<p>
+  {% blocktrans %}
+    Best regards,
+  {% endblocktrans %}
+</p>
+
+<p>
+  {% blocktrans %}The CommCare HQ Team{% endblocktrans %}<br />
+  {{ base_url }}
+</p>
```

Comment:
```suggestion
</p>

```
nit new line at end of file